kong/clustering.lua

```diff
@@ -454,25 +455,42 @@ function _M.init_worker(conf)
     assert(shdict, "kong_clustering shdict missing")
 
     -- ROLE = "control_plane"
+    local db = kong.db
 
     kong.worker_events.register(function(data)
       -- we have to re-broadcast event using `post` because the dao
       -- events were sent using `post_local` which means not all workers
       -- can receive it
+
+      -- send event to other workers in the same node
```

Comment:
It sends it to all workers in node, including itself.

Reply:
Yes, fixing my comment